src/entry-io.js

```diff
@@ -78,7 +78,7 @@ class EntryIO {
         return
       }
 
-      return new Promise(async (resolve, reject) => {
+      return new Promise((resolve, reject) => {
```

Comment:
`standard` doesnt like async promise executors anymore. So I had to refactor this and lines 140-158 below

Reply:
I'm wondering if it might be better to move all of the previous logic into a separate async function inside the callback as i believe that is valid standardjs. The only big difference I can see between the diffs is that the fromMultihash promise was caught before.

Reply:
Added a rejection at the end of that block, good catch.